### PR TITLE
Adjustments to remove some chpl unstable warnings

### PR DESCRIPTION
**Pull request by jeremiah-corrado**
Makes some small code adjustments to remove a few common unstable warnings:

* warning: 'localSubdomains' on arrays is unstable and may change in the future
* warning: The module name 'AutoMath' is unstable.  If you want to use qualified naming on the symbols within it, please 'use' or 'import' the Math module
* warning: calling the 'this' method explicitly is unstable and may change in the future
* warning: symbol names with leading underscores ( ) are unstable
* warning: symbol names beginning with 'chpl_' ( ) are unstable

### CODE REVIEW

**Review by jaketrookman — APPROVED** [review]

File: src/comp.out
Comment (pierce314159) [inline src/comp.out:1]: What purpose does this file serve? I'm guessing it wasn't meant to be committed

**Review by pierce314159 — COMMENTED** [review]
Other than questions about the `comp.out` file, this looks good!

File: src/comp.out
Comment (jeremiah-corrado) [reply src/comp.out:1]: Oops, this was an accidental commit. Thanks!

**Review by jaketrookman — APPROVED** [review]

**Review by pierce314159 — APPROVED** [review]
Looks good to me!

**Comment by pierce314159** [comment]
@jeremiah-corrado once you could resolve the conflicts, this should be good to merge

**Pull request merged**